src/editor.js

```diff
@@ -478,6 +567,24 @@ class Editor {
     assert(false, `I don't know how to toggle "${name}".`);
   }
 
+  async loadHtml_({target: {dataset}}) {
+    const name = assert(dataset.name);
+    const contentResponse = await fetch(
+      (await this.state_.templates)[name].contentUrl
+    );
+    assert(contentResponse.status == 200);
+    this.codeMirror_.setValue(await contentResponse.text());
+    this.state_.files = (await this.state_.templates)[name].assets.map(
```

Comment:
1. Let's use `files` instead of `assets` and destructure for style:
2. Let's define a class method to get template URLs.

```js
async selectTemplate_({target: {dataset}}) {
  // ...
  const {files} = (await this.state_.templates)[name];
  this.state_.files = files.map(filename => ({
    name: filename,
    url: this.getTemplateFileUrl_(name, filenanme),
  });
  // ...
}

getTemplateFileUrl_(templateName, filename) {
  return `${this.baseUrlPrefix_}/static/templates/${templateName}/${filename}`;
}
```